app/src/main/java/org/fossasia/openevent/app/module/attendee/list/AttendeesFragment.java

```diff
@@ -107,13 +110,36 @@ public void onCreate(Bundle savedInstanceState) {
 
     public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         inflater.inflate(R.menu.menu_attendees, menu);
+
         MenuItem search = menu.findItem(R.id.search);
         searchView = (SearchView) search.getActionView();
         setupSearchListener();
+
+        MenuItem filterPendingSync = menu.findItem(R.id.pendingSync);
+        filterPendingSync.setChecked(false);
+
         super.onCreateOptionsMenu(menu, inflater);
     }
 
     @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
+            case R.id.pendingSync:
+                item.setChecked(!item.isChecked());
+                if (item.isChecked()) {
+                    item.getIcon().setTint(utilModel.getResourceColor(R.color.color_accent));
+                    fastItemAdapter.filter(IS_SYNC_PENDING);
+                } else {
+                    item.getIcon().setTint(Color.BLACK);
+                    fastItemAdapter.filter("");
+                }
```

Comment:
Rather than doing this, add a sort popup dialog which will contain items by which the attendees can be sorted. It will only contain one entry for now -> Pending

In future, it can contain other's like attendees with paid tickets only, or checked in attendees / checked out attendees, etc